beacon-chain/blockchain/process_block_test.go

```diff
@@ -2446,3 +2446,281 @@ func fakeResult(missing []uint64) map[uint64]struct{} {
 	}
 	return r
 }
+
+func TestSaveLightClientUpdate(t *testing.T) {
+	//if !features.Get().EnableLightClient {
+	//	return
+	//}
```

Comment:
You can't check if a feature is enabled inside a test because it would mean that the test doesn't run at all when the feature is turned off by default. There is a way to turn a feature on for one test, but we can update this in a later PR.